ruma/Cargo.toml

```diff
@@ -17,6 +17,7 @@ either = ["ruma-identifiers/either"]
 rand = ["ruma-identifiers/rand"]
 unstable-pre-spec = ["ruma-client-api/unstable-pre-spec"]
 unstable-synapse-quirks = ["ruma-client-api/unstable-synapse-quirks"]
+unstable-exhaustive-types = ["ruma-events/unstable-exhaustive-types", "ruma-appservice-api/unstable-exhaustive-types", "ruma-client-api/unstable-exhaustive-types", "ruma-federation-api/unstable-exhaustive-types"]
```

Comment:
This line is quite long. Can you put each item on its own line (with the last one also having a trailing comma)?